observation/calibrate_delays.py

```diff
@@ -67,8 +67,9 @@ class NoTargetsUpError(Exception):
         target = sources_above_horizon.targets[0]
         target.add_tags('bfcal single_accumulation')
         session.capture_init()
-        user_logger.info("Only calling capture_start on correlator stream directly")
-        session.cbf.correlator.req.capture_start()
+        user_logger.info("Only calling capture_start on correlator streams")
+        for correlator in session.cbf.correlators:
```

Comment:
Since starting data capture for all correlators would be the normal thing to do, should we rather have the loop in session?